### ISSUE TITLE
[Bug]: insertContent(' ') does not write a space into the editor

### ISSUE THREAD

**Issue by Chaddeus** [ueberdosis/tiptap#4833]
### Which packages did you experience the bug in?

core

### What Tiptap version are you using?

2.1.16

### What’s the bug you are facing?

This previously worked. I updated TipTap and now it appears maybe there is some sort of `.trim()` or space removal when using `editor.insertContent()`

For example:

```
editor.chain().insertContent(' ').scrollIntoView().run()
```

Does not write the space into the editor. Which is a massive impact on my app that streams tokens into the editor from AI -- as occasionally a token can be a space (such as before numbers).

### What browser are you using?

Chrome

### What did you expect to happen?

I would expect `editor.chain().insertContent(' ').scrollIntoView().run()` to write a space into the editor.

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by svenadlung** [comment]
Agree, I was able to replicate and localize the problem. Will be working on a fix.

In the meantime, and as a quick fix for your current situation (expecting strings), can you go with this?

```editor.chain().focus().insertContent({
  type: 'text',
  text: ' ',
}).run()```

**Comment by Chaddeus** [comment]
> Agree, I was able to replicate and localize the problem. Will be working on a fix.
> 
> In the meantime, and as a quick fix for your current situation (expecting strings), can you go with this?
> 
> ```
> editor.chain().focus().insertContent({
>   type: 'text',
>   text: ' ', // your string/char
> }).run()
> ```

Thank you, this will work until the update. Really appreciate the quick response. 🙌
